petastorm/spark/spark_dataset_converter.py

```diff
@@ -283,7 +305,15 @@ def make_spark_converter(
     else:
         compression_codec = "uncompressed"
 
+    # pylint: disable=global-statement
+    global _cache_format
+    # pylint: enable=global-statement
+    if _is_delta_available():
+        _cache_format = "delta"
+    else:
+        _cache_format = "parquet"
```

Comment:
Move 
```
    if _is_delta_available():
        _cache_format = "delta"
    else:
        _cache_format = "parquet"
```
into global level.
When the spark_converter package init, the `_cache_format` will be init and won't be changed again.

Reply:
Oh, I am wrong. The parquet and delta test may be run in a same processs.
So we always get `_cache_format` in runtime. 
Remove `_cache_format` var, and add a global method like:
```
def get_cache_format():
    if _is_delta_available():
        return "delta"
    else:
        return "parquet"
```